wayfire.ini

```diff
@@ -132,14 +135,11 @@ outputs = kanshi
 notifications = mako
 
 # Screen color temperature
-# http://jonls.dk/redshift/
-# Note: Requires Redshift with Wayland support.
-# https://github.com/jonls/redshift/pull/663
-redshift = redshift -m wayland
+# https://sr.ht/~kennylevinsen/wlsunset/
+gamma = wlsunset
 
 # Idle configuration
 # https://github.com/swaywm/swayidle
-# https://github.com/swaywm/swaylock
```

Comment:
Why is this lined removed?

Reply:
you can set up swayidle from a config file now, plus there's a separate section for swaylock

Reply:
well the example itself uses swaylock so I think the link should remain there.